packages/types/Calendar.d.ts

```diff
@@ -115,6 +115,13 @@ export interface BookingLimit {
   PER_YEAR?: number | undefined;
 }
 
+export interface DurationLimit {
+  PER_DAY?: number | undefined;
+  PER_WEEK?: number | undefined;
+  PER_MONTH?: number | undefined;
+  PER_YEAR?: number | undefined;
+}
+
```

Comment:
Same goes here - this type is the same as `BookingLimit` maybe we should do the same here too :)